loops-arrays-objects.js

```diff
@@ -32,8 +34,26 @@ const scores = [
 // D
 // ==========================================
 
-
-
+for (let i = 0; i < scores.length; i++) {
+    // console.log(scores[i].score);
+    switch (true) {
```

Comment:
in uitwerking NOVI kiezen ze voor if -else if statements. Jouw manier met switch statement bij 1b en 1c werkt ook prima en kan ook prima.